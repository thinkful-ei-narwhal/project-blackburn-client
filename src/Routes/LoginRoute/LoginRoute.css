.Login {
  text-align: center;
}
.log-form {
  display: flex;
  align-items: center;
  flex-direction: column;
  justify-content: space-around;
  margin: 10px auto;
  padding: 20px;
  width: 50%;
}
.log-btn {
  margin: 10px auto;
  padding: 5px;
}
.log-input {
  margin: 10px;
}

.error {
  color: red;
  font-size: 15px;
  font-weight: normal;
}
