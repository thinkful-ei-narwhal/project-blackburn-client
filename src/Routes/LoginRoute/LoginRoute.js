import React, { Component } from 'react';
import './LoginRoute.css';
import LoginForm from '../../Components/LoginForm/LoginForm';

class LoginRoute extends Component {
  render() {
    return (
      <div className="Login">
        <h2>Login:</h2>
        <LoginForm />
      </div>
    );
  }
}

export default LoginRoute;
