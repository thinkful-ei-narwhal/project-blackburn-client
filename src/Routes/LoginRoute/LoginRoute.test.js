import React from 'react';
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import BlackBurnContext from '../../Context/BlackburnContext';
import LoginRoute from './LoginRoute';

describe(`Renders login page to dom without crashing`, () => {
    it('renders checkpoint win crashing', () => {
        const wrapper = shallow(
        <BlackBurnContext.Provider>
            <LoginRoute />
        </BlackBurnContext.Provider>
    )
    .find('.Login')
    expect(toJson(wrapper)).toMatchSnapshot()
    })
})