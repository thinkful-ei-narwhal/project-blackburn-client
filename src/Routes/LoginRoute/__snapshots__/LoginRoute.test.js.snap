// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders login page to dom without crashing renders checkpoint win crashing 1`] = `null`;
